km/km_vcpu_run.c

```diff
@@ -495,7 +495,9 @@ static void km_vcpu_one_kvm_run(km_vcpu_t* vcpu)
    vcpu->regs_valid = 0;                            // Invalidate cached registers
    vcpu->sregs_valid = 0;
    km_infox(KM_TRACE_VCPU, "about to ioctl( KVM_RUN )");
+   errno = 0;
```

Comment:
Is this really needed?

Reply:
ioctl doesn't set it if there is no error. So the code below would act on old value